tests/conftest.py

```diff
@@ -222,6 +221,24 @@ def running_time_entry(
     )
 
 
+@pytest.fixture(scope="module")
+def activity_repository(app: Flask) -> ActivityCosmosDBRepository:
+    with app.app_context():
+        from commons.data_access_layer.cosmos_db import init_app, cosmos_helper
+
+        if cosmos_helper is None:
+            init_app(app)
+
+    return ActivityCosmosDBRepository()
+
+
+@pytest.fixture
+def activities_dao():
```

Comment:
We are not using the activities dao,we may remove it.

Reply:
I just deleted it